elephant/asset.py

```diff
@@ -1687,7 +1687,7 @@ def cluster_matrix_entries(mask_matrix, eps, min_neighbors, stretch):
         mask_matrix : np.ndarray
             The boolean matrix, whose elements with positive values are to be
             clustered. The output of :func:`ASSET.mask_matrices`.
-        eps : float
+        max_distance : float
```

Comment:
if the max distance is denoted as :math:`\eps` in the paper, it's always a good practice to relate the argument/variable names to the math names in the paper
but this is a minor thing and even if it's the case in asset, I'll not pursue it here.